CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.1.0
 * New features (API):
+  * `xOptimize` normalization pass added to remove undefined alternatives from
+    case expressions, see [#918](https://github.com/clash-lang/clash-compiler/pull/935)
```

Comment:
I think I'd put this under "Internal changes"

Reply:
See I did think that, but it does technically change the API of `Clash.Normalize.Transformations` by exporting something new. I can change it if you want?

Reply:
Ah right, we don't consider `clash-lib` a public API. Only `clash-prelude` is.

Reply:
Gotcha, I'll move it then... and wait another hour for CI :cry: